modules/tinymce/src/themes/silver/main/ts/ui/menus/contextmenu/platform/MobileContextMenu.ts

```diff
@@ -101,8 +91,26 @@ const setupiOSOverrides = (editor: Editor) => {
   };
 };
 
-const show = (editor: Editor, e: EditorEvent<TouchEvent>, items: MenuItems, backstage: UiFactoryBackstage, contextmenu: AlloyComponent, useNodeAnchor: boolean, highlightImmediately: boolean) => {
-  const anchorSpec = useNodeAnchor ? getNodeAnchor(editor) : getPointAnchorSpec(editor, e);
+const getAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>, anchorType: Coords.AnchorType) => {
+  const anchorSpec = Coords.getAnchorSpec(editor, e, anchorType);
+  if (anchorType === 'point') {
+    return {
+      bubble: Bubble.nu(0, bubbleSize, bubbleAlignments),
+      layouts,
+      overrides: {
+        maxWidthFunction: MaxWidth.expandable(),
+        maxHeightFunction: MaxHeight.expandable()
+      },
+      ...anchorSpec
+    };
+  } else {
+    return anchorSpec;
+  }
+};
```

Comment:
```suggestion
const getAnchorSpec = (editor: Editor, e: EditorEvent<TouchEvent>, anchorType: Coords.AnchorType) => {
  const anchorSpec = Coords.getAnchorSpec(editor, e, anchorType);
  return {
    bubble: Bubble.nu(0, 0, bubbleAlignments),
    layouts,
    overrides: {
      maxWidthFunction: MaxWidth.expandable(),
      maxHeightFunction: MaxHeight.expandable()
    },
    ...anchorSpec
  };
};
```

As mentioned on slack, after looking at this more and playing around with the branch locally I believe this is what we want. At the very least the overrides should apply to `selection` and `point`.

Note: I've also removed the `bubbleSize` variable in the above as we don't render the "bubble arrow" on mobile so I don't think we want the context toolbar to be sitting 12px below the selection/touch point.